rosdep/base.yaml

```diff
@@ -2850,6 +2850,34 @@ libopenvdb-dev:
   fedora: [openvdb-devel]
   gentoo: [media-gfx/openvdb]
   ubuntu: [libopenvdb-dev, libilmbase-dev]
+liborocos-bfl:
+  debian:
+    '*': [liborocos-bfl0.8]
+    'stretch': null
+  fedora: [liborocos-kdl]
```

Comment:
Should this point to a `*-bfl` package instead of `*-kdl`?

Reply:
Good catch